### PR TITLE
Latest boms

### PR DESCRIPTION
**Pull request by peter-lawrey**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -15,18 +15,19 @@
...
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
 
     <parent>
         <groupId>net.openhft</groupId>
         <artifactId>java-parent-pom</artifactId>
-        <version>1.1.29</version>
-        <relativePath />
+        <version>1.24.0-SNAPSHOT</version>
```
Comment (alamar) [inline pom.xml:24]: I don't believe we should set it to SNAPSHOT or releasing would fail

File: pom.xml [hunk 2]
```diff
@@ -15,18 +15,19 @@
...
+        <relativePath/>
     </parent>
 
     <modelVersion>4.0.0</modelVersion>
     <artifactId>chronicle-ticker</artifactId>
-     <version>2.23ea2-SNAPSHOT</version>
+    <version>2.24ea1-SNAPSHOT</version>
```
Comment (alamar) [inline pom.xml:30]: Makes sense

File: pom.xml [hunk 3]
```diff
@@ -39,12 +40,24 @@
     </properties>
 
     <profiles>
-       <profile>
-           <id>java11</id>
-           <properties>
-               <makefile>Makefile11</makefile>
-           </properties>
-       </profile>
+        <profile>
```
Comment (alamar) [inline pom.xml:43]: Could be done, but I've just brought `java11` profile back

File: pom.xml [hunk 4]
```diff
@@ -39,12 +40,24 @@
...
+            <id>java11</id>
+            <activation>
+                <jdk>[11,16]</jdk>
+            </activation>
+            <properties>
+                <makefile>Makefile</makefile>
```
Comment (alamar) [inline pom.xml:49]: Should be Makefile11?

File: pom.xml [hunk 5]
```diff
@@ -39,12 +40,24 @@
...
+            </properties>
+        </profile>
+        <profile>
+            <id>java17</id>
+            <activation>
+                <jdk>[17,</jdk>
+            </activation>
+            <properties>
+                <makefile>Makefile</makefile>
```
Comment (alamar) [inline pom.xml:58]: Should be Makefile17?

File: pom.xml [hunk 6]
```diff
@@ -53,7 +66,7 @@
                 <groupId>net.openhft</groupId>
                 <artifactId>third-party-bom</artifactId>
                 <type>pom</type>
-                <version>3.19.2</version>
+                <version>3.24.0-SNAPSHOT</version>
```
Comment (alamar) [inline pom.xml:69]: Would likely cause release to fail due to SNAPSHOT here

File: src/main/c/Makefile17 [hunk 7]
```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/make -f
+#
+# Makefile for C code
```
Comment (alamar) [inline src/main/c/Makefile17:3]: I don't believe we have build on Java17 currently (we test Java17 against code compiled by Java 8) so I'm not sure we would be able to test this.

**Review by RobAustin — COMMENTED** [review]
Ilya has already commented on this

**Review by RobAustin — CHANGES_REQUESTED** [review]
tests are failing 
image

**Comment by alamar** [comment]
I believe we should archive this change for later, in case we ever need it.

**Comment by peter-lawrey** [comment]
This has been replaced by some work Ilya did.

**Pull request closed without merging**